400-claims.md

```diff
@@ -120,6 +123,10 @@ The parameter data stored in a claim data is _the resolved key/value pairs_ that
   - Valid user-supplied values are presented
   - Default values are supplied for all parameters where `default` is provided and no user-supplied value overrides this
 
+### Outputs
+
+The output data is a map from the name of each output as defined in the bundle to the content of the file located at the path for that output at the end of the action.
```

Comment:
I think an example here would help the reader, as the sentence is quite complex to parse 😅

Reply:
@astrieanna could you add an example to this PR and then we can merge this once everyone re-reviews?